rest/changes_api.go

```diff
@@ -419,6 +419,13 @@ func (h *handler) sendSimpleChanges(channels base.Set, options db.ChangesOptions
 					}
 					_ = encoder.Encode(entry)
 					lastSeq = entry.Seq
+					revid, readAudit := entry.NeedsDocReadAudit()
```

Comment:
This code shows up in a few places, maybe could be a function on changeEntry like entry.AuditDocRead(ctx)?